### PR TITLE
Endpoint/Controller/Service for the cluster dropdown.

### PR DESCRIPTION
**Pull request by dkirillov**
@BrunoCarrier

### CODE REVIEW

File: gate-core/src/main/groovy/com/netflix/spinnaker/gate/services/internal/ClouddriverService.groovy [hunk 1]
```diff
@@ -286,4 +281,9 @@ interface ClouddriverService {
   List<Map> getRoles(@Path("cloudProvider") String cloudProvider,
                      @Path("account") String account,
                      @Path("region") String region)
+
+  @GET('/ecsClusters/{cloudProvider}/{account}/{region}')
```
Comment (BrunoCarrier) [inline gate-core/src/main/groovy/com/netflix/spinnaker/gate/services/internal/ClouddriverService.groovy]: No need for {cloudProvider} as this is ecs-specific.  Unless AWS folks did the same for their aws cluster controller? @dkirillov

File: gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/EcsClusterController.groovy [hunk 2]
```diff
@@ -0,0 +1,38 @@
+/*
+  * Copyright 2014 Netflix, Inc.
```
Comment (BrunoCarrier) [inline gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/EcsClusterController.groovy]: you work at netflix? :D

File: gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/EcsClusterController.groovy [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping("/ecsClusters")
+class EcsClusterController {
+  @Autowired
+  EcsClusterService ecsClusterService
+
+  @ApiOperation(value = "Retrieve a list of ECS clusters that can be used by the selected cloud provider")
+  @RequestMapping(value = "/{provider}/{account}/{region}", method = RequestMethod.GET)
```
Comment (BrunoCarrier) [inline gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/EcsClusterController.groovy]: I don't think other cloud providers can use these ECS clusters

File: gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/EcsClusterController.groovy (hunk 2 above)
Comment (dkirillov) [reply gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/EcsClusterController.groovy]: Copy pasted :D now I guess I do.

Comment (dkirillov) [reply gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/EcsClusterController.groovy]: Removing it altogether - unless we have our own correct/legal one...

Comment (BrunoCarrier) [reply gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/EcsClusterController.groovy]: Just replace `Netflix` with `Lookout`

File: gate-core/src/main/groovy/com/netflix/spinnaker/gate/services/internal/ClouddriverService.groovy (hunk 1 above)
Comment (dkirillov) [reply gate-core/src/main/groovy/com/netflix/spinnaker/gate/services/internal/ClouddriverService.groovy]: Removed.

File: gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/EcsClusterController.groovy (hunk 3 above)
Comment (dkirillov) [reply gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/EcsClusterController.groovy]: Fixed.
The path no longer needs `/{provider}/`.

**Comment by BrunoCarrier** [comment]
LGTM +2

**Pull request merged**